### ISSUE TITLE
error while joining 

### ISSUE THREAD

**Issue by CraftYourWay** [ST-DDT/CrazyLogin#3]
getting this error while joining
````
[15:28:50 ERROR]: Could not pass event PlayerLoginEvent to CrazyLogin v7.23.4
org.bukkit.event.EventException
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[lobby.jar:git-Spigot-a9631d2-67b2424]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[lobby.jar:git-Spigot-a9631d2-67b2424]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.PlayerList.attemptLogin(PlayerList.java:507) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.LoginListener.b(LoginListener.java:118) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.LoginListener.E_(LoginListener.java:54) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.NetworkManager.a(NetworkManager.java:233) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.ServerConnection.c(ServerConnection.java:140) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:832) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:399) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:673) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:572) [lobby.jar:git-Spigot-a9631d2-67b2424]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
Caused by: java.lang.NoSuchMethodError: org.bukkit.Bukkit.getOnlinePlayers()[Lorg/bukkit/entity/Player;
        at de.st_ddt.crazyplugin.CrazyPlayerDataPlugin.getOnlinePlayersPerIP(CrazyPlayerDataPlugin.java:234) ~[?:?]
        at de.st_ddt.crazylogin.listener.PlayerListener.PlayerLoginConnectionCheck(PlayerListener.java:158) ~[?:?]
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_91]
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_91]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_91]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_91]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[lobby.jar:git-Spigot-a9631d2-67b2424]
        ... 13 more
````

**Comment by ST-DDT** [comment]
Which version of CrazyCore are you using?
This looks like an issue that I have fixed in latest CrazyCore.

**Comment by CraftYourWay** [comment]
this one:

R: CrazyLogin v7.23.4 for 1.10 Jun 29, 2016

**Comment by ST-DDT** [comment]
Thanks for this report.
This seems to be a new change.
Fixed in: [CrazyCore v10.7.9](http://dev.bukkit.org/bukkit-plugins/crazycore/files/100-crazy-core-v10-7-9/) 
I haven't had time to test CrazyCore and CrazyLogin thoroughly yet, so if you encounter any bugs please tell me.
